lib/storage/metadata/mongoclient/MongoClientInterface.ts

```diff
@@ -1682,7 +1686,7 @@ class MongoClientInterface {
                 // a master or re-delete it in between so place an
                 // atomic condition on the PHD flag and the mst
                 // version:
-                // eslint-disable-next-line
+                 
```

Comment:
uneeded empty line (between a comment and the code it refers to) + space at end of line

```suggestion
```